### PR TITLE
Fix some format

### PR DESCRIPTION
**Pull request by stewart-yu**
Fix format

> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Allow edits from maintainers checkbox

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/5996)

### CODE REVIEW

File: docs/getting-started-guides/libvirt-coreos.md [hunk 1]
```diff
@@ -46,8 +46,8 @@ On the other hand, `libvirt-coreos` might be useful for people investigating low
 4. Install [libvirt](http://libvirt.org/)
 5. Install [openssl](http://openssl.org/)
 6. Enable and start the libvirt daemon, e.g.:
-   * ``systemctl enable libvirtd && systemctl start libvirtd`` # for systemd-based systems
-   * ``/etc/init.d/libvirt-bin start`` # for init.d-based systems
+   * ``systemctl enable libvirtd && systemctl start libvirtd``  for systemd-based systems
+   * ``/etc/init.d/libvirt-bin start``  for init.d-based systems
```
Comment (heidecke) [inline docs/getting-started-guides/libvirt-coreos.md]: These two lines show up as numbered list (1, 2) and not bullets in the preview.

Comment (stewart-yu) [reply docs/getting-started-guides/libvirt-coreos.md]: It's normal, origin web show up as bullets in the preview also, but will show up as numbered list (1, 2) in webpage.

Comment (abiogenesis-now) [reply docs/getting-started-guides/libvirt-coreos.md]: would prefer if these were bullet points---can you try putting a new line before these bullet points? sometimes that fixes the markdown

Comment (steveperry-53) [reply docs/getting-started-guides/libvirt-coreos.md]: @abiogenesis-now Hmm. I tried to get this inner list to be bulleted. See the two most recent commits.But it still renders as numbered.

**Comment by steveperry-53** [comment]
@abiogenesis-now I tried getting the inner list to be bulleted. Can't seem to get it to work. See the two most recent commits.

**Comment by stewart-yu** [comment]
It's so strange.

**Comment by heidecke** [comment]
@stewart-yu @steveperry-53 did you try four spaces before the bullet? It should be similar to:

1. numbered item 1
    * nested bullet 1
    * nested bullet 2
1. numbered item 2
1. numbered item 3

```
1. numbered item 1
    * nested bullet 1
    * nested bullet 2
1. numbered item 2
1. numbered item 3
```

**Comment by steveperry-53** [comment]
@heidecke Yes, we've tried four spaces.
I'm going to say we've spent enough time in this. Merging.
If any of you feel strongly that we should stick with this bulleted list problem, feel free to open another PR, and we can fiddle with it there.

**Pull request merged**